eldritch/values.py

```diff
@@ -382,6 +382,51 @@ def annotation(self, state):
     return f"{spend_min}-{spend_max} {self.SPEND_TYPES[self.spend_type]}"
 
 
+class FlexibleRangeSpendPrerequisite(SpendValue):
+  def __init__(self, spend_types: list, spend_min: float, spend_max: float):
+    assert all(spend_type in self.SPEND_TYPES for spend_type in spend_types)
+    super().__init__()
+    self.spend_types_ = spend_types
```

Comment:
Maybe `_spend_types` instead of `spend_types_`? The latter is almost never seen in python.

Reply:
Can you tell I've been doing a lot of [sklearn](https://stackoverflow.com/questions/49138692/what-does-it-mean-for-an-attribute-name-to-end-in-an-underscore) in my day job? Will fix.

Reply:
I can't tell, but only because I've not heard of sklearn until now.